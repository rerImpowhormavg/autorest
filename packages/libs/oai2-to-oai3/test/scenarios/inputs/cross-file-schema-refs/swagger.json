{
  "$schema": "https://github.com/OAI/OpenAPI-Specification/blob/master/schemas/v2.0/schema.json",
  "swagger": "2.0",
  "info": {
    "x-ms-metadata": {
      "apiVersions": ["test-0.1"]
    },
    "title": "Test",
    "description": "test",
    "version": "test-0.1"
  },
  "paths": {},
  "definitions": {
    "MySchema": {
      "type": "object",
      "properties": {
        "myProp": { "$ref": "other.json#/definitions/MySharedSchema" }
      }
    }
  }
}
