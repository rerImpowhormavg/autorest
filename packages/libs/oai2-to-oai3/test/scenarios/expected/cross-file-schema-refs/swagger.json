{
  "servers": [],
  "$schema": "https://github.com/OAI/OpenAPI-Specification/blob/master/schemas/v2.0/schema.json",
  "openapi": "3.0.0",
  "info": {
    "x-ms-metadata": {
      "apiVersions": [
        "test-0.1"
      ]
    },
    "title": "Test",
    "description": "test",
    "version": "test-0.1"
  },
  "paths": {},
  "components": {
    "schemas": {
      "MySchema": {
        "type": "object",
        "properties": {
          "myProp": {
            "$ref": "other.json#/components/schemas/MySharedSchema"
          }
        }
      }
    }
  }
}